appservice/src/FileUtilities.ts

```diff
@@ -53,8 +53,7 @@ export async function zipDirectory(folderPath: string): Promise<string> {
         zipper.pipe(zipOutput);
         zipper.glob('**/*', {
             cwd: folderPath,
-            dot: true,
-            ignore: 'node_modules{,/**}'
+            dot: true
```

Comment:
Kudu requires you to zip up the node modules too?

Reply:
They don't require it, but the default behavior of zipdeploy assumes a ready-to-run app. See [here](https://github.com/projectkudu/kudu/wiki/Deploying-from-a-zip-file) for more info, specifically in the part about SCM_DO_BUILD_DURING_DEPLOYMENT.

We could go against the default behavior and set  SCM_DO_BUILD_DURING_DEPLOYMENT to true, but I didn't think that was worth the added complexity. For example, do we overwrite the user's existing app setting if they have one? Do we put the app setting in a '.deployment' file? What if they already have a '.deployment' file?

Also - what if they don't have a standard nodejs project? Maybe they have a dotnet project with random files in a 'node_modules' folder. Maybe they have a nodejs project, but they install npm packages to a different folder.

I'm interested in what other people think, but I see my PR description about how zip deploy now supports larger files. I didn't think it was as bad to include the modules directly.

Reply:
Discussed offline. I will keep this PR as-is and address at a later time. See issue #7